GOVERNANCE.md

```diff
@@ -166,19 +166,18 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
+If the private discussion receives some support (in form of explicit comments or
```

Comment:
Can we make this explicit and say that at least one +1

```suggestion
If the private discussion receives at least one +1 from another collaborator (in form of explicit comments or
```

Reply:
I've decided to not try to define what support should be, I think the informal thing is enough given it's an optional step anyway. I'd expect the "I'm going to open the public issue in 72 hours" be enough if folks think there hasn't been enough support

Reply:
I will rise it a bit (maybe +5 or +10?) so far we have 85 collaborators listed ([ref](https://github.com/nodejs/node#collaborators).